### PR TITLE
Hide the skeleton in the closet

### PR DESCRIPTION
**Pull request by loic-sharma**

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[7db4c1b](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1391/commits/7db4c1bf7bacb4118415a04caecad5245291fad3)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/api/repository-signatures-resource.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-7/docs/api/repository-signatures-resource.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/api/repository-signatures-resource?branch=pr-en-us-1391) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C8%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201904082232044975-1391%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=31cSboX3yeyvdzp%2FcS4WFt2P6psHJ44FdrF3S9B8IsU%3D&st=2019-04-08T22%3A28%3A25Z&se=2019-05-09T22%3A33%3A25Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/api/repository-signatures-resource.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ The following `@type` value is used:
 @type value                | Notes
 -------------------------- | -----
 RepositorySignatures/4.7.0 | The initial release
-RepositorySignatures/4.9.0 | Allows enabling `allRepositorySigned`
+RepositorySignatures/5.0.0 | Allows enabling `allRepositorySigned`
```
Comment (joelverhagen) [inline docs/api/repository-signatures-resource.md]: Did an official client ship with support for 4.9.0?

If so, I personally we feel we should admit that it exists. It's a pivot available to us in the future if desired.

Comment (loic-sharma) [reply docs/api/repository-signatures-resource.md]: Done

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[6beca01](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1391/commits/6beca01ff2ee68845ea7530d83dc6d91af57e105)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/api/repository-signatures-resource.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-7/docs/api/repository-signatures-resource.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/api/repository-signatures-resource?branch=pr-en-us-1391) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C8%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201904082306593465-1391%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=%2B0gCHvnkOD%2FH23X8YLVCSlwd%2BpbO9KJ%2FTxhpNFtBv6E%3D&st=2019-04-08T23%3A03%3A22Z&se=2019-05-09T23%3A08%3A22Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**